### DISCUSSION TITLE
DiagnosticReport

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#22] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [DiagnosticReport](http://hl7.org/fhir/R4/diagnosticreport.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core DiagnosticReport, or if there is a need for an additional DiagnosticReport profile in AU Core.

AU Core DiagnosticReport profile would set minimum expectations for a DiagnosticReport resource to record, search, and fetch diagnostic reports associated with a patient. 

AU Core DiagnosticReport should be based on the [AU Base DiagnosticReport](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-diagnosticreport.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the DiagnosticReport resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by xtfer** [comment]
This would imply the presence of Specimen and ImagingStudy, to be useful. Do they need to be added?

**Comment by mjosborne1** [comment]
Diagnostic report relies on the Accession Number profile: https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-accessionnumber.html . In that profile, Accession Type is a CodableConcept required binding inherited from AU Base, the two values for which are ASCN : Accession Identifier and LACSN: A laboratory accession id.
I don't see the point of making these a required binding type, they are practically useless information without having types for  other reports e.g. Diagnostic Imaging accession no, cardiology investigation accession number, lung function accession ....
In my mind LACSN is-a ASCN and hence they are interchangeable.
I would either make the type field useful or make it non-mandatory.

**Reply by dt-r** [reply]
This is an AU Base comment.

**Reply by mjosborne1** [reply]
I feel like we are missing a discussion forum (like this one) in AU Base. My only options are to mention it here, Zulip or log an issue in AU Base. I don't feel strongly enough about it for it to be an actual Issue, so I withdraw the comment.

**Reply by dt-r** [reply]
I think this question is best for Zulip. The AU Base authors and others can see the query around the Accession Identifier profile there and respond to answer your query.

**Comment by davidmckillop-ADHA** [comment]
### DiagnosticReport using a panel of Observations (as per EU) or not (as per US)

There has been an ongoing discussion in various Zulip threads about the representation of Observations in a DiagnosticReport i.e whether there should be a "grouper" Observation to represent a panel of Observations or not.  Angus's diagram (5th June 2023 comment) from [Usage of panel of Observations](https://chat.fhir.org/#narrow/stream/179256-Orders-and-Observation-WG/topic/Usage.20of.20panel.20of.20Observations) best illustrates the issue with the representation of a Full Blood Count (FBC) as an example:
![image](https://github.com/hl7au/au-fhir-core/assets/19895989/ae7fc2fc-8c3e-4ce6-bbfa-a2daa47c384b)
HL7 EU have adopted option A where as the US favour option B.
There were a series of meetings with HL7 O&O in June and July refer to [7th June](https://confluence.hl7.org/display/OO/2023-06-07+OO+on+FHIR+-+Adhoc+-+Panel+of+OBS+Discussion), [14th June](https://confluence.hl7.org/display/OO/2023-06-14+OO+on+FHIR+-+Adhoc+-+Panel+of+OBS+Discussion).  I'm sure there were 3 or 4 meetings, but that's all I could find the minutes for.  
A lot was discussed at the meetings, including the ramifications of the 2 different options/models, but nothing was agreed about which option, or if any option, should be chosen.
HL7 O&O will be discussing [Panel - Composition - DiagnosticReport](https://confluence.hl7.org/pages/viewpage.action?pageId=197330693#id-20231019Main-Panel-Composition-DiagnosticReport) in mid November, but that time could be used to discuss the response to the recently released [HL7 EU Laboratory Report FHIR Implementation Guide ballot](https://chat.fhir.org/#narrow/stream/179256-Orders-and-Observation-WG/topic/HL7.20EU.20Laboratory.20Report.20FHIR.20Implementation.20Guide.20ballot).

[HL7 AU Base](https://build.fhir.org/ig/hl7au/au-fhir-base/) and [HL7 AU Core](https://build.fhir.org/ig/hl7au/au-fhir-core/) have recommended grouping of Observations (option A) with the profile specific implementation guidance of:
_When constructing a report for a study / panel:
the individual component examinations are referenced by that grouping Observation in Observation.hasMember and not directly referenced in DiagnosticReport.result
DiagnosticReport.code and the study / panel Observation Observation.code should be the same concept if the report contains only the results of that study / panel_

This thread is to inform the AU community of the ongoing international discussions and if/when any further details are forthcoming Angus or I will update this thread.
However, in the meantime for those in the diagnostics space your input on [HL7 AU Base](https://build.fhir.org/ig/hl7au/au-fhir-base/) and [HL7 AU Core](https://build.fhir.org/ig/hl7au/au-fhir-core/) would be appreciated, and please feel free to provide feedback on the [HL7 EU Laboratory Report FHIR Implementation Guide ballot](https://chat.fhir.org/#narrow/stream/179256-Orders-and-Observation-WG/topic/HL7.20EU.20Laboratory.20Report.20FHIR.20Implementation.20Guide.20ballot).

**Reply by MattCordell** [reply]
I think both those patterns are valid... for different reasons in different circumstances.
(I note [Observation](https://www.hl7.org/fhir/observation.html) supports a single, or a group too).
1st reason is aesthetics.
E.g. For a Full Blood Count, you might have the counts/indices in one section. And a second for the (optional) differential.
I've seen paper based FBCs based as both Option A or B.

2nd Reason is relating results. The U MC/S is a good example.
The micro "section" is probably aesthetics. But if two organisms are identified, the sensitivities should relate to each..
I _think_ ,from memory, in v2, each organism is treated as a specimen, and reference by the different sensitivities.E.g. Organism1, Organism2.

I can see a few different ways we could do this, but an agreed, consistent approach would be good. Are there any existing examples @davidmckillop ? (ie. MCS results for 2+ organisms?)

**Comment by davidmckillop-ADHA** [comment]
Hi @MattCordell  - we're on the same page in that [Observation](https://www.hl7.org/fhir/observation.html) supports both with different approaches being taken by different regions.

Re: representation of multiple organisms in a HL7 V2 microbiology report - that's dealt with by OBX-4 (sub-id) refer to example in section [4.4.2.4 OBX-4 Observation sub-ID (ST) 00572](https://hl7.org.au/archive/hl7v2wg/4-Observation-Reporting_1278278.html#id-4ObservationReporting-4.4.2.4OBX-4Observationsub-ID(ST)00572) of the Australian Diagnostics and Referral Messaging - Localisation of HL7 Version 2.4 
(Note: the "regular" Confluence url for this standard localisation is no longer working; hence this "archive" url.  Query part of the issues with the HL7 AU transition from HL7 AU Confluence to HL7 International infrastructure).

We haven't developed any specific multiple organism microbiology reports as yet, though it would follow the following pattern of:
![Urine-overall](https://github.com/hl7au/au-fhir-core/assets/19895989/51cf28b5-322a-4bfb-9a49-43579617710f)

Note: this is not a standard format, some sites will have less elements or major groups e.g. without the "Dipstick" content and some may have more elements.  
Using the above pattern, option A would look like:
![Urine-optionA](https://github.com/hl7au/au-fhir-core/assets/19895989/6efc41dd-5418-4e4f-a799-76d7c6f54c5d)

And option B would look like:
![Urine-optionB](https://github.com/hl7au/au-fhir-core/assets/19895989/49738d4b-1a48-446b-b602-ad7e8c573763)


There's not a lot of difference between the two i.e only the additional grouper Observation (line 2 in option A) and the subsequent additional Observation in each row.

I agree that there are a few different ways that this could done, and an agreed, consistent approach would be good.

It will be useful to know any decisions that come from the HL7 International O&O meetings (scheduled for mid Nov) as this may provide guidance on how AU progress on this topic.  However, if there isn't any international guidance forthcoming, then the HL7 AU community will need to decide on the best way forward which may be Option A, or Option B or living with both with guidance on how to use both or some other option.
